scripts/verify-deployment.ts

```diff
@@ -0,0 +1,65 @@
+import { execSync } from 'child_process';
+import yargs from 'yargs';
+
+type ProcessArgs = {
+  version: string;
+  isPrerelease: boolean;
+};
+const githubBinaries = ['amplify-pkg-linux-arm64.tgz', 'amplify-pkg-linux.tgz', 'amplify-pkg-macos.tgz', 'amplify-pkg-win.exe.tgz'];
+const parseArgs = async (): Promise<ProcessArgs> => {
+  const args = yargs(process.argv.slice(2))
+    .version(false)
+    .options({
+      v: { alias: 'version', type: 'string', demandOption: true },
+      p: { alias: 'prerelease', type: 'boolean', default: false },
```

Comment:
```suggestion
      gh: { alias: 'include-github', type: 'boolean', default: false },
```

To de-confuse naming.